script.js

```diff
@@ -1,57 +1,83 @@
-const add = document.getElementById('add');
-const awesomeBooks = [];
+const StoredBooks = [];
 
-add.addEventListener('click', (e) => {
-  e.preventDefault();
-  const title = document.getElementById('title');
-  const author = document.getElementById('author');
+class Display {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
 
-  const book = {
-    Title: title.value,
-    Author: author.value,
-  };
-  const list = document.getElementById('book-list');
-  const newDiv = document.createElement('div');
-  newDiv.classList.add('book');
+  static addtolist() {
+    const Title = document.getElementById('title').value;
+    const Author = document.getElementById('author').value;
+    const list = document.getElementById('book-list');
+    const newDiv = document.createElement('div');
+    newDiv.classList.add('book');
+    const newbook = new Display(Title, Author);
+    StoredBooks.push(newbook);
```

Comment:
- [ ] (Optional) It would be nice to use the `concat` method instead of the `push` method to remove a book. That way, you do not mutate the original array. It is also considered good practice to not mutate arrays when changing states. 👍